CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [Unreleased][]
+
+[Unreleased]: https://github.com/chaostoolkit-incubator/chaostoolkit-datadog/compare/main...jitapichab:chaostoolkit-datadog:feature/datadog_metrics
+
+
```

Comment:
Probably not what you wanted to do here. In the general case, it's very good you add an entry to the changelog but you don't need to set a new version as I'll do it on release. Please remove this header and instead add a changelog entry under the existing `[Unreleased][]` block.